terraform_compliance/main.py

```diff
@@ -57,7 +59,8 @@ def cli(arghandling=ArgHandling(), argparser=ArgumentParser(prog=__app_name__,
                 '--write-steps-once',
                 features_directory,
                 '--basedir', steps_directory,
-                '--user-data=plan_file={}'.format(args.plan_file)]
+                '--user-data=plan_file={}'.format(args.plan_file),
+                '--junit-xml={}'.format(args.junit_file)]
```

Comment:
We don't want this, since these are default parameters that has been passed to `radish`, and `--junit-xml` is not a default value for `terraform-compliance` nor `radish`.

Please remove this.

Reply:
I misunderstood the usage. Thanks for the detail, I've reverted this change.